python/ray/_private/external_storage.py

```diff
@@ -251,7 +251,9 @@ class FileSystemStorage(ExternalStorage):
             spill objects doesn't exist.
     """
 
-    def __init__(self, directory_path, buffer_size=None):
+    def __init__(
+        self, directory_path, node_id=None, use_node_id_subdir=None, buffer_size=None
+    ):
```

Comment:
Keep `node_id` and `use_node_id_subdir` outside of this class. They are not related to storage and purely application logic.

Reply:
Removed `use_node_id_subdir`.
At the caller side all these args are expanded from a dict:
```
_external_storage = FileSystemStorage(**config["params"])
```
So I kept `node_id` for now. Let me know if you prefer a different way.